bower.json

```diff
@@ -19,6 +19,7 @@
     "locales": "https://github.com/citation-style-language/locales.git#d2b612f8a6f764cbd66e67238636fac3888a7736",
     "raven-js": "2.1.0",
     "zeroclipboard": "2.1.6",
-    "osf-style": "https://github.com/CenterForOpenScience/osf-style.git#5dcf5997ad2808ffc7c8564f94cde102bf7a4a96"
+    "osf-style": "https://github.com/CenterForOpenScience/osf-style.git#5dcf5997ad2808ffc7c8564f94cde102bf7a4a96",
+    "At.js": "https://github.com/CenterForOpenScience/At.js.git#d4c1c6ebce3808a275bac59740dabd9bcf76d96f"
```

Comment:
Good that this is under COS now. Let's put a notice at the top of the README on master explaining that it's a fork and the rationale for forking, including a link to https://github.com/ichord/At.js/pull/394.